src/pages/library.tsx

```diff
@@ -103,30 +92,8 @@ function Library() {
                         <div className="library-movie">
                             {isLoading ? 'Loading....' :
                                 movies.map((element) => (
-                                    <Card sx={{ maxWidth: 300, mt: '20px', bgcolor: '#212E53' }}>
-                                        <CardHeader sx={{ color: 'white' }}
-                                            title={element.title}
-                                            subheader={formatDate(element.release_date)}
-                                        />
-                                        <CardMedia
-                                            component="img"
-                                            height="194"
-                                            image='https://quai10-website.s3.eu-west-3.amazonaws.com/backgrounds/sonic-3-le-film-0-fe033741ae88fb6d9e5296f7efd19e5c-0_2024-12-23-152750_vtuh.jpg'
-                                            alt={element.title}
-                                        />
-                                        <CardContent>
-                                            <Typography variant="body2" sx={{ color: 'gray' }}>
-                                                {element.overview}
-                                            </Typography>
-                                        </CardContent>
-                                        <CardActions disableSpacing >
-                                            <Typography sx={{ color: 'white' }}>
-                                                {element.vote_average}/10
-                                            </Typography>
-                                        </CardActions>
-                                    </Card>
+                                    renderCards(true, element)
```

Comment:
renderCards est un composant React, tu peux donc l'appeler comme d'hab
```suggestion
<RenderCards
    showClassic
    movie={element}
/>
```